### PR TITLE
Manual release: publish to master

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Seems like the publish to master flow is not really fixed yet for some reason ([here](https://github.com/luniehq/lunie/runs/818760410?check_suite_focus=true) the logs), so we will need to do this the old way.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: api/data/network-capabilities.js [hunk 1]
```diff
@@ -141,6 +141,23 @@ const getNetworkCapabilities = {
...
   },
+  'polkadot-testnet': {
+    feature_session: capabilityEnum.ENABLED,
+    feature_explore: capabilityEnum.ENABLED,
+    feature_portfolio: capabilityEnum.ENABLED,
+    feature_validators: capabilityEnum.ENABLED,
+    feature_proposals: capabilityEnum.DISABLED,
+    feature_activity: capabilityEnum.DISABLED,
+    feature_explorer: capabilityEnum.DISABLED,
+    action_send: capabilityEnum.ENABLED,
```
Comment (jbibla) [inline api/data/network-capabilities.js:152]: are transfers enabled on polkadot yet?

File: api/data/network-coinlookups.js [hunk 2]
```diff
@@ -100,6 +100,11 @@ const coinLookupDictionary = {
       viewDenom: 'DOT',
       chainToViewConversionFactor: 1e-12
   }],
+  "polkadot-testnet": [{
+    chainDenom: 'Planck',
```
Comment (jbibla) [inline api/data/network-coinlookups.js:104]: is this for real?

File: api/data/networks.js [hunk 3]
```diff
@@ -196,32 +197,58 @@ module.exports = [
...
-  //   icon:
-  //     '/img/networks/polkadot-mainnet.png',
-  //   slug: 'polkadot',
-  //   lockUpPeriod: '28 days'
-  // },
+  {
+    id: 'polkadot',
+    title: 'Polkadot',
+    chain_id: 'polkadot-cc1',
+    api_url: 'https://api-01.polkascan.io/polkadot/api/v1/',
```
Comment (jbibla) [inline api/data/networks.js:204]: what are we using polkascan api for?

File: api/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "api",
-  "version": "1.0.28",
+  "version": "1.0.30",
```
Comment (jbibla) [inline api/package.json:3]: skipping 29?

File: api/data/network-capabilities.js (hunk 1 above)
Comment (Bitcoinera) [reply api/data/network-capabilities.js:152]: yes, they are. Actually this is outdated. Now even staking is enabled

File: api/package.json (hunk 4 above)
Comment (Bitcoinera) [reply api/package.json:3]: I think there was an api release that was skipped (PR was closed)? I am not sure. But good point

File: api/data/networks.js (hunk 3 above)
Comment (Bitcoinera) [reply api/data/networks.js:204]: currently for nothing at all

**Pull request closed without merging**